src/pages/ProfileInfoPage.jsx

```diff
@@ -0,0 +1,208 @@
+  },
+  {
+    name: "LinkedIn Profile",
+    icon: faLinkedin,
+    key: "linkedinUrl",
+    bg_color: "EC0707",
+    color: "0A48E0",
+  },
+  {
+    name: "YouTube Profile",
+    icon: faYoutube,
+    key: "youtubeUrl",
+    bg_color: "red",
+    color: "white",
+  },
+];
+
+const ProfileInfoPage = () => {
+  const inputStyle = "p-2 border-2 border-current md:p-4 focus:outline-none";
+  const iconStyle = "w-3 h-3 mx-2 text-lg text-gray-500 md:h-5 md:w-5";
+
+  const [formState, setFormState] = useState(DEFAULT_DATA);
+
+  const handleChange = (event, key) => {
+    setFormState({ ...formState, [key]: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(formState);
+  };
+
+  return (
+    <section className="relative pb-8 border border-b-0 border-gray-400">
+      <div className="flex flex-wrap justify-between gap-2 p-4">
+        <div className="flex">
+          <img src={profileImg} alt="profile image" className="w-20 mr-3" />
+          <div>
+            <h1 className="font-bold">Ahmed Ashraf</h1>
+            <p className="text-gray-500">No risk No Fun</p>
+            <p className="text-gray-500">Jop tittle</p>
+          </div>
+        </div>
+        <div>
+          <p>
+            <FontAwesomeIcon icon={faBookOpen} className={iconStyle} />
+            <span className="font-bold text-blue-600"> 4 Learning</span>
+          </p>
+          <p>
+            <FontAwesomeIcon icon={faArchive} className={iconStyle} />
+            <span className="font-bold text-blue-600"> 2 Achieved</span>
+          </p>
+        </div>
+      </div>
+      <div className="absolute w-full border-b border-gray-400"></div>
+      <form
+        className="grid grid-cols-12 gap-6 px-3 py-10 font-bold leading-8 md:text-xl md:px-5 lg:px-10"
+        onSubmit={handleSubmit}
+      >
+        {/* First Name input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-6`}
+          placeholder="First Name "
+          value={formState.firstName}
+          onChange={(e) => handleChange(e, "firstName")}
+        />
+
+        {/* Last Name input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-6`}
+          placeholder="Last Name"
+          value={formState.secondName}
+          onChange={(e) => handleChange(e, "secondName")}
+        />
+
+        {/* Description input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12`}
+          placeholder="Description"
+          value={formState.description}
+          onChange={(e) => handleChange(e, "description")}
+        />
+
+        {/* Job input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-7 lg:col-span-8`}
+          placeholder="Add your job (Student)"
+          value={formState.jop}
+          onChange={(e) => handleChange(e, "jop")}
```

Comment:
Take care of English words spelling, it's "Job" not "Jop".

I would recommend changing this function name to be more clear, instead of "handleChange", it could be for example "handleStoringUserData"